### ISSUE TITLE
Switch the UI Style to Qt Fusion [Suggestion]

### ISSUE THREAD

**Issue by konero** [opentoonz/opentoonz#1635]
I know generally features are not posted here, but let me explain anyway. I'm the one who wrote the current themes which uses external QSS.

My problem with it currently is somewhere down the line there was a big reliance placed on QSS to fix and workaround a bunch of grid issues, instead of fixing those issues in the code directly, I was really puzzled when I came across them.

Unfortunately, I haven't got time to do this, but I thought I'd post this suggestion incase another developer wants to give it a go someday.

I'd like to strongly suggest switching the style to Qt Fusion (as a default). Why? Mainly because it's low maintenence and widely documented. It's dead simple to use by pretty much a single line of code, since it's bundled with Qt5, and it can be easily switched while in runtime. Creating alternate color palettes is also a lot easier, with much easier to understand nomenclature.

If any developer is interested, there is a good tutorial and project you can download on the EVILEG blog which explains how to change styles in runtime, how to create a color palette and how to install the palette.

https://evileg.com/en/post/239/

Basically, just this.

```C++
void Widget::on_darkPushButton_clicked()
{
    // Create a palette for a dark theme
    QPalette darkPalette;

    // Customize the color palette for the interface elements
    darkPalette.setColor(QPalette::Window, QColor(53, 53, 53));
    darkPalette.setColor(QPalette::WindowText, Qt::white);

    // Install this palette
    qApp->setPalette(darkPalette);
}
```

This reduces (official) overhead. Additional theme styles would still be possible, and those could be maintained by the community akin to Blender, Krita, etc.

It will probably require a lot of refactoring for all the custom components, but TBH there isn't that many bar the XSheet, Function Editor, etc.

Anyway, that's my suggestion.

**Comment by turtletooth** [comment]
I actually much prefer the current themes to Fusion.  I think you have done a great job on the themes, and since they are already made, I don't see a whole lot of work needed in maintaining them, especially compared to the work that would be required to refactor the custom widgets to work well with fustion.
